pkg/controllers/update.go

```diff
@@ -65,7 +65,13 @@ func RunInstallScript() (bool, string, Error) {
 		fmt.Println(strOut)
 	}
 	if err != nil {
-		return false, "", Error{Err: err, Message: "Unable to install the latest Doppler CLI"}
+		message := "Unable to install the latest Doppler CLI"
+		// check for errors indicating lack of perms
+		if strings.Contains(strOut, "dpkg: error: requested operation requires superuser privilege") {
```

Comment:
When this error occurs, does dpkg return an EACCES (13) error code? It might be a bit cleaner to check that instead, if possible. Otherwise, I think the error message check is better than nothing 👍

Reply:
That's a good point. The tricky thing is that that error would be captured inside the `install.sh` script, but that error code isn't bubbled up. I think for now this might be the easiest solution.